.gitignore

```diff
@@ -13,3 +13,4 @@ yarn-error.log*
 /.changelog
 .npm/
 yarn.lock
+.eslintcache
```

Comment:
Can you revert? We're going to update this save location in another PR

Reply:
Of course, I was just trying to figure it out 😄